pom.xml

```diff
@@ -53,7 +53,7 @@
           </dependency>
```

Comment:
Please check the profile (line 15). You are using the @Tag, not the @Category. As I know, I won't work.
If it works. Please add the command to run them in the readme file

Reply:
Yes it does not work, and I think it would be better if I delete the command in the readme file